nullaway/src/test/java/com/uber/nullaway/jspecify/JSpecifyVarargsTests.java

```diff
@@ -596,7 +595,7 @@ public void varargsOverride() {
             "    }",
             "    static class NullableVarargsBothImpl3 implements NullableVarargsBoth {",
             "        @Override",
-            "        // BUG: Diagnostic contains: Parameter has type Object[]",
+            "        // BUG: Diagnostic contains: Parameter has type Object @Nullable []",
```

Comment:
Does the diagnostic say Object @Nullable now because the default behavior is that un-annotated parameters are nullable?

Reply:
No, we just weren't printing the type properly at all before.  If we have a varargs parameter `Object @Nullable... params`, the type of the `params` array is `Object @Nullable []`.  We're just fixing a serious bug in printing types in error messages.